package.json

```diff
@@ -8,10 +8,10 @@
     "build-demo": "mkdir -p dist && npm run dist-prod && cp dist/kinto.min.js demo/kinto.js",
     "demo": "npm run build-demo && http-server demo",
     "dist": "mkdir -p dist && rm -f dist/*.* && npm run dist-dev && npm run dist-prod && npm run dist-noshim && npm run dist-fx",
-    "dist-dev": "browserify --ignore process -s Kinto -d -e src/index.js -o dist/kinto.js -t [ babelify --sourceMapRelative . ]",
-    "dist-noshim": "browserify --ignore process -s Kinto -g uglifyify --ignore isomorphic-fetch --ignore babel-polyfill -e src/index.js -o dist/kinto.noshim.js -t [ babelify --sourceMapRelative . ]",
-    "dist-prod": "browserify --ignore process -s Kinto -g uglifyify -e src/index.js -o dist/kinto.min.js -t [ babelify --sourceMapRelative . ]",
-    "dist-fx": "BABEL_ENV=firefox browserify -s loadKinto --bare --ignore uuid --ignore kinto-http --ignore isomorphic-fetch -e fx-src/index.js -o temp.jsm -t [ babelify --sourceMapRelative . ] && mkdir -p dist && cp fx-src/jsm_prefix.js dist/moz-kinto-offline-client.js && echo \"\n/*\n * Version $npm_package_version - $(git rev-parse --short HEAD)\n */\n\" >> dist/moz-kinto-offline-client.js && cat temp.jsm >> dist/moz-kinto-offline-client.js && rm temp.jsm",
+    "dist-dev": "BABEL_ENV=development browserify --ignore process -s Kinto -d -e src/index.js -o dist/kinto.js -t [ babelify --sourceMapRelative . ]",
+    "dist-noshim": "BABEL_ENV=development browserify --ignore process -s Kinto -g uglifyify --ignore isomorphic-fetch --ignore babel-polyfill -e src/index.js -o dist/kinto.noshim.js -t [ babelify --sourceMapRelative . ]",
+    "dist-prod": "BABEL_ENV=development browserify --ignore process -s Kinto -g uglifyify -e src/index.js -o dist/kinto.min.js -t [ babelify --sourceMapRelative . ]",
```

Comment:
Also, I wonder about a dedicated `noshim` .babelrc environment, where we wouldn't enable any transform; would that work?